docs/source/en/api/pipelines/stable_diffusion/control_net.mdx

```diff
@@ -0,0 +1,94 @@
+|[takuma104/control_sd15_scribble](https://huggingface.co/takuma104/control_sd15_scribble)<br/> *Trained with human scribbles*  |A hand-drawn monochrome image with white outlines on a black background.|<a href="https://huggingface.co/takuma104/controlnet_dev/blob/main/gen_compare/control_images/converted/control_vermeer_scribble.png"><img width="64" src="https://huggingface.co/takuma104/controlnet_dev/resolve/main/gen_compare/control_images/converted/control_vermeer_scribble.png"/></a>|
+|[takuma104/control_sd15_seg](https://huggingface.co/takuma104/control_sd15_seg)<br/>*Trained with semantic segmentation*  |An [ADE20K](https://groups.csail.mit.edu/vision/datasets/ADE20K/)'s segmentation protocol image.|<a href="https://huggingface.co/takuma104/controlnet_dev/blob/main/gen_compare/control_images/converted/control_room_seg.png"><img width="64" src="https://huggingface.co/takuma104/controlnet_dev/resolve/main/gen_compare/control_images/converted/control_room_seg.png"/></a>|
+
+
+## Resources
+
+- [Colab Notebook Example](https://colab.research.google.com/drive/1AiR7Q-sBqO88NCyswpfiuwXZc7DfMyKA?usp=sharing)
+- [controlnet_hinter](https://github.com/takuma104/controlnet_hinter): Image Preprocess Library for ControlNet
+
+## Usage example
+
+- Basic Example (Canny Edge)
+```python
+from diffusers import StableDiffusionControlNetPipeline
+from diffusers.utils import load_image
+
+# Canny edged image for control
+canny_edged_image = load_image("https://huggingface.co/takuma104/controlnet_dev/resolve/main/vermeer_canny_edged.png")
+pipe = StableDiffusionControlNetPipeline.from_pretrained("takuma104/control_sd15_canny").to("cuda")
+image = pipe(prompt="best quality, extremely detailed", image=canny_edged_image).images[0]
+image.save("generated.png")
+```
+
+- Using SD1.x variant model to control
+```py
+from diffusers import StableDiffusionControlNetPipeline, AutoencoderKL, UNet2DConditionModel
+from diffusers.utils import load_image
+
+# Canny edged image for control
+canny_edged_image = load_image("https://huggingface.co/takuma104/controlnet_dev/resolve/main/vermeer_canny_edged.png")
+
+base_model_id = "prompthero/openjourney"  # an example: openjourney model
+vae = AutoencoderKL.from_pretrained(base_model_id, subfolder="vae").to("cuda")
+unet = UNet2DConditionModel.from_pretrained(base_model_id, subfolder="unet").to("cuda")
+
+pipe = StableDiffusionControlNetPipeline.from_pretrained("takuma104/control_sd15_canny", unet=unet, vae=vae).to("cuda")
+image = pipe(prompt="best quality, extremely detailed", image=canny_edged_image, width=512, height=512).images[0]
+image.save("generated.png")
+```
+
```

Comment:
Let's show the result here if we can, so the reader can see the difference.

Reply:
I agree! I tried using the same seed and settings with OpenJourney, but the differences were not as clear as I had expected. I also tried with other models, but only Anything, the anime model, showed clear differences. I thought Anything would become more clearly anime-like, but it wasn't quite so. Maybe it's a characteristic of the Canny model, so I'm thinking of generating images with other ControlNet models to see if there are differences.

https://huggingface.co/takuma104/controlnet_dev/blob/main/model_compare/README.md

Reply:
The `Anything` model is clearly different, yes. Another option could be https://huggingface.co/22h/vintedois-diffusion-v0-1 (haven't tested it myself).

Reply:
Looks like a nice model! I will try this one as well.

Reply:
I have added `vintedois-diffusion` as a new Unet model. It may not have as distinctive features as Anything, but it still stands out.
https://huggingface.co/takuma104/controlnet_dev/blob/main/model_compare/README.md

I tried other ControlNet models to see if the Canny model had strong constraints on the image. Looking at these results, Depth and OpenPose seem to be good options. If you want to use Vermeer's painting, OpenPose's pose estimation may not be very effective, so Depth may be the best option. What do you think?
https://huggingface.co/takuma104/controlnet_dev/blob/main/model_compare_full/README.md

`Normal` looks a little strange because the RGB conversion failed.